pkg/ingester/activeseries/active_series_test.go

```diff
@@ -414,15 +420,15 @@ func BenchmarkActiveSeries_UpdateSeries(b *testing.B) {
 			const nLabels = 10
 			builder := labels.NewScratchBuilder(nLabels)
 			series := make([]labels.Labels, tt.nSeries)
-			hash := make([]uint64, tt.nSeries)
+			refs := make([]uint64, tt.nSeries)
 			for s := 0; s < tt.nSeries; s++ {
 				builder.Reset()
 				for i := 0; i < nLabels; i++ {
 					// Label ~20B name, ~40B value.
 					builder.Add(fmt.Sprintf("abcdefghijabcdefghi%d", i), fmt.Sprintf("abcdefghijabcdefghijabcdefghijabcd%d", s))
```

Comment:
Since this benchmark isn't using matchers, I think it doesn't matter which labels each series has anymore.